backend/src/jwt/decode.js

```diff
@@ -7,7 +7,7 @@ export default async (driver, authorizationHeader) => {
   try {
     const decoded = await jwt.verify(token, process.env.JWT_SECRET)
     id = decoded.sub
-  } catch {
+  } catch (e) {
```

Comment:
Just as a consistency hint: e stands often for event, try to use err as an error shorthand.